packages/deploy-tooling/test/unit/cli/config.test.ts

```diff
@@ -17,14 +17,16 @@ describe('cli/config', () => {
 
     describe('getDeploymentConfig', () => {
         test('valid config path', async () => {
-            expect(await getDeploymentConfig(join(__dirname, '../../test-input/ui5-deploy.yaml'))).toBeDefined();
+            expect(
+                await getDeploymentConfig(join(__dirname, '../../fixtures/simple-app/ui5-deploy.yaml'))
+            ).toBeDefined();
         });
         test('invalid config', async () => {
-            await expect(getDeploymentConfig(join(__dirname, '../../test-input/ui5.yaml'))).rejects.toThrowError();
+            await expect(getDeploymentConfig(join(__dirname, '../../simple-app/ui5.yaml'))).rejects.toThrowError();
         });
         test('invalid path', async () => {
             await expect(
-                getDeploymentConfig(join(__dirname, '../../test-input/ui5-invalid.yaml'))
+                getDeploymentConfig(join(__dirname, '../../simple-app/ui5-invalid.yaml'))
```

Comment:
Same as above. But actually we can keep this since we are testing for an invalid path.

Reply:
Fixed, see above.